patroni/validator.py

```diff
@@ -937,6 +937,18 @@ def validate_watchdog_mode(value: Any) -> None:
 schema = Schema({
     "name": str,
     "scope": str,
+    Optional("log"): {
+        Optional("level"): EnumValidator(('DEBUG', 'INFO', 'WARN', 'WARNING', 'ERROR', 'FATAL', 'CRITICAL'),
+                                         case_sensitive=True, raise_assert=True),
+        Optional("traceback_level"): EnumValidator(('DEBUG', 'ERROR'), raise_assert=True),
```

Comment:
why is this one case-insensitive?

Reply:
Because it performs case-insensitive checks here: https://github.com/zalando/patroni/blob/c1ee99d81da8b35e370350a433c5f5c4889baff5/patroni/log.py#L253-L258